apps/homework/tasks.py

```diff
@@ -14,41 +15,63 @@
 
 logger = logging.getLogger(__name__)
 
+
 class SubmissionPostException(BaseException):
 
     def __init__(self, *args, **kwargs):
         self.sub_type = kwargs.get('sub_type')
         super(SubmissionPostException, self).__init__(*args, **kwargs)
 
+
+def retrieve_score_from_jupyter_notebook(fd):
+    content = fd.readlines()
+    filtered_content = list(filter(lambda line: 'final score' in str(line) and 'print' not in str(line), content))
+    if len(filtered_content) is not 1:
```

Comment:
I think `!=` is more appropriate here, `is` also checks (or should) the reference of the object to make sure they're the same.. think Ted or Chris and I played with this recently and was surprised to see `is` not behave exactly like we expected

http://net-informations.com/python/iq/is.htm

Reply:
At the expense of Chagrade's record-setting performance, I concede. ✅